### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by LightRuby** [cupiditys/iReady-Overload#285]
Can you make it to where my school doesn't find out about it? Because they did.

**Comment by comradeduck** [comment]
Confused. Just hide the bookmark or copy and delete it if they check

**Comment by LightRuby** [comment]
> Confused. Just hide the bookmark or copy and delete it if they check

nononononono. They FOUND out. And EVERYONE who ever used it got lunch detention.

**Comment by X2isme** [comment]
> > Confused. Just hide the bookmark or copy and delete it if they check
> 
> nononononono. They FOUND out. And EVERYONE who ever used it got lunch detention.

do you know how they found out?

**Comment by LightRuby** [comment]
Yep. Some idiotic person put it on Tik-Tok and I guess they asked I-Ready about it and they went in and found out.

**Comment by cupiditys** [comment]
were people using it at school? if so theres your answer why

**Comment by LightRuby** [comment]
No. There was some people but they never found out until the tik tok.

**Comment by cupiditys** [comment]
i mean the people who got caught

**Comment by LightRuby** [comment]
Some people did. Others didn't.

**Comment by lechickene** [comment]
i think his school monitors all computers

**Comment by comradeduck** [comment]
> Yep. Some idiotic person put it on Tik-Tok and I guess they asked I-Ready about it and they went in and found out.

Exactly and that's what made everything public

**Comment by cupiditys** [comment]
theyve known it for like 4 months not just cuz some dude on tiktok posted it

**Comment by LightRuby** [comment]
srsly....?
